qiskit_addon_cutting/qpd/instructions/qpd_gate.py

```diff
@@ -160,13 +154,15 @@ def __init__(
         basis_id: int | None = None,
         label: str | None = None,
     ):
-        """
-        Initialize the single qubit QPD gate, and assign member variables.
+        """Initialize the single qubit QPD gate, and assign member variables.
 
         Args:
+            basis: TODO
             qubit_id: This gate's relative index to the decomposition which it belongs.
                 Single qubit QPDGates should have qubit_id 0 if they describe a local
                 decomposition, such as a wire cut.
+            basis_id: TODO
+            label: TODO
```

Comment:
These need docstring

Reply:
Users might sometimes witness this object, but they should basically never be calling this constructor.  I can add a few descriptions here, but even better would be to make the "here be dragons" nature clear somehow.

Reply:
Makes sense. You can add a warning to the docstring above `Args`. https://sublime-and-sphinx-guide.readthedocs.io/en/latest/notes_warnings.html

Reply:
Fixed in 5192ba782ff0b9c9b37830401f0ea4e461a3b2b7.